Server/include/shutdown.h

```diff
@@ -16,6 +16,8 @@
 #ifndef __SHUTDOWN_H__
 #define __SHUTDOWN_H__
 
+#include <uavariant.h>
```

Comment:
Is this include needed? I didn't see any new references to variant types in the changes for this file.

Reply:
ok, fair enough